web_view_calendar_list/static/src/js/calendar_list_renderer.js

```diff
@@ -0,0 +1,154 @@
+odoo.define('web_view_calendar_list.CalendarListRenderer', function (require) {
+    "use strict";
+
+    var CalendarRenderer = require('web.CalendarRenderer');
+    var session = require('web.session');
+    var core = require('web.core');
+    var qweb = core.qweb;
+    var _t = core._t;
+
+    var scales = {
+        day: 'listDay',
+        week: 'listWeek',
+        month: 'listMonth'
+    };
+
+    var AppointmentRenderer= CalendarRenderer.extend({
+        _initCalendar: function () {
+            var self = this;
+
+            this.$calendar = this.$(".o_calendar_widget");
+
+            // This seems like a workaround but apparently passing the locale
+            // in the options is not enough. We should initialize it beforehand
+            var locale = moment.locale();
+            $.fullCalendar.locale(locale);
+
+            //Documentation here : http://arshaw.com/fullcalendar/docs/
+            var fc_options = $.extend({}, this.state.fc_options, {
+                eventDrop: function (event) {
+                    self.trigger_up('dropRecord', event);
+                },
+                eventResize: function (event) {
+                    self.trigger_up('updateRecord', event);
+                },
+                eventClick: function (event) {
+                    self.trigger_up('openEvent', event);
+                    self.$calendar.fullCalendar('unselect');
+                },
+                select: function (target_date, end_date, event, _js_event, _view) {
+                    var data = {'start': target_date, 'end': end_date};
+                    if (self.state.context.default_name) {
+                        data.title = self.state.context.default_name;
+                    }
+                    self.trigger_up('openCreate', data);
+                    self.$calendar.fullCalendar('unselect');
+                },
+                eventRender: function (event, element) {
+                    var $render = $(self._eventRender(event));
+                    event.title = $render.find('.o_field_type_char:first').text();
+                    element.find('.fc-list-item-title').html($render.html());
+                    element.addClass($render.attr('class'));
+                    var display_hour = '';
+                    if (!event.allDay) {
+                        var start = event.r_start || event.start;
+                        var end = event.r_end || event.end;
+                        var timeFormat = _t.database.parameters.time_format.search("%H") != -1 ? 'HH:mm': 'h:mma';
```

Comment:
can't you use format utils?

Reply:
The field is a datetime, and no parser exists on field_utils in order to parse datetime to time. I am following the same behavior found on odoo. It might not be the best, but I tried to follow the same logic in order to not make much changes.